code/obj/item/clothing/masks.dm

```diff
@@ -320,6 +323,22 @@
 			return 1
 		return 0
 
+	attack_self(mob/user as mob)
+		if(!src.mask_bald)
+			src.mask_bald = TRUE
+			src.name = "wigless clown mask"
+			src.desc = bald_desc_state
+			src.icon_state = bald_icon_state
```

Comment:
since all the bald icons follow the pattern `[normal icon_state]_bald`, just set the state to `"[src.icon_state]_bald"` here. Then you can remove the `bald_icon_state` var entirely.

Reply:
done